src/main/java/TaskManager.java

```diff
@@ -0,0 +1,83 @@
+public class TaskManager {
+    public static final int MAX_TASKS = 100;
+    public static final int TODO_TASK_NAME_BEGIN_INDEX = 5;
+    public static final int DEADLINE_TASK_NAME_BEGIN_INDEX = 9;
+    public static final int SKIP_BY_IN_DEADLINE = 4;
+    public static final int EVENT_TASK_NAME_BEGIN_INDEX = 6;
+    public static final int SKIP_FROM_IN_EVENT = 6;
+    public static final int SKIP_TO_IN_EVENT = 4;
```

Comment:
I'm not sure if I like these constant names because the names do not give a clear idea of what they are used for.

Reply:
Changed constant names to be clearer